test/harness/jpype/types/Outer.java

```diff
@@ -0,0 +1,15 @@
+package jpype.types;
+
+public interface Outer
+{
+  Object get();
+
+  public class Inner implements Outer
```

Comment:
programmers who would ever use such a coding style should be tared and feathered :)

Reply:
It seems to come up pretty often with interfaces where an interface contains a collection of concrete implementations.  Certainly not something that I endorse as I never managed to trip this bug in in the 100 repos that I work on (spent the week converting from bitbucket to gitlab and the system admin through it was some sort of joke when we pushed 100 repos in), but am not sure dropping a coder into boiling tar is a good solution.  I have seen much, much worse styles than this.

Fun fact, did you know that this code will compile in Java...
```
  public static void main(String[] args)
  {
    List List = new ArrayList();
    List.add(1);
  }
```